Tensile/Source/client/source/Reference.cpp

```diff
@@ -605,8 +669,9 @@ namespace Tensile
                                       << spatialCoord[1] << "," << spatialCoord[0]
                                       << " dIndex=" << dIndex << " value=" << value << "\n";
                         }
-                        inputs.d[dIndex] = static_cast<typename Inputs::DType>(inputs.alpha)
-                                           * static_cast<typename Inputs::DType>(value);
+                        inputs.d[dIndex] = static_cast<typename Inputs::DType>(
+                            multiply<typename Inputs::AlphaType, Accumulator, Inputs, Accumulator>(
```

Comment:
can we simply use alphatype for alpha computing??

static_cast<typename Inputs::DType>(
inputs.allpha * static_cast<typename Inputs::AlphaType>(value)
)

Reply:
Using alphatype here should against the case  H/H/H + HPA
In this case, the type of rC is float and alphtype is half.
this is also the reason for not using DType.